### DISCUSSION TITLE
How many github repos do I have?

### DISCUSSION THREAD

**Discussion by stormasm** [community/community#22100] [New to GitHub]
<p>With the UI redesign I can no longer see the number of github repos I have…</p>
<p>It used to be at the top of the page next to the repositories tab with a<br>
little number showing the number of repos you have…</p>
<p>Now that number seems to have disappeared ?</p>
<p>Where is the number to show me how many repos I have ?</p>
<p>Thank you !</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/stormasm">@stormasm</a>!</p>
<p>Thank you so much for reaching out here about that change.  You are correct, there presently is not a way to view those repository counts at this time in the WebUI.</p>
<p>I will share your feedback and experience with our product team to look into. While we can’t promise that your feature request will be offered in the future, your request is certainly in the right place.</p>
<p>If you are interested, as an alternative, you can use the REST API to help generate that information. You can read more about it from the two links below. The first is general information and usage of the API, and the second is the specific help page for repository information.</p>
<ul>
<li><a href="https://developer.github.com/v3/">REST API v3</a></li>
<li><a href="https://developer.github.com/v3/repos/#list-repositories-for-the-authenticated-user">List repositories for the authenticated user</a></li>
</ul>

**Comment by stormasm** [comment, marked as answer]
<p>Great news !  Github fixed this issue and we are now able to once<br>
again see how many repos we have on Github.</p>
<p>Thank you Github for such a quick turn around time on this issue<br>
that was broken for a few weeks.</p>
